salt/modules/powerdns.py

```diff
@@ -0,0 +1,567 @@
+"""
+Create, modify and delete PowerDNS zone
+"""
+
+import logging
+import socket
+
+import requests
```

Comment:
This needs to be in a try/except since `requests` is not a hard dependency of salt.
As an alternative use `__utils__["http.query"]` or `salt.utils.http.query()` directly.

Reply:
Added exception handler.